### DISCUSSION TITLE
[공통 컴포넌트] Card

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#8] [General]
<img width="195" alt="스크린샷 2022-06-07 오후 4 43 32" src="https://user-images.githubusercontent.com/72402747/172324744-431b2d8a-f958-4110-8978-d0ce0f14da5e.png">

**Comment by NamgyungKim** [comment]
Post {
            "likes": Like[],
            "comments": Comment[],
            "_id": String,
            "image": Optional<String>,
            "imagePublicId": Optional<String>,
            "title": String,
            "channel": Channel,
            "author": User,
            "createdAt": String,
            "updatedAt": String
         }
